Assets/LeapMotionModules/Interaction/Examples/Buttons/InteractionSlider.cs

```diff
@@ -1,173 +1,84 @@
-﻿using Leap.Unity.GraphicalRenderer;
-using UnityEngine;
-using UnityEngine.Events;
-
+﻿using UnityEngine;
+using Leap.Unity.Attributes;
 namespace Leap.Unity.UI.Interaction {
-  /** A physics-enabled button. Activation is triggered by physically pushing the button back to its compressed position. 
-   *  Increasing the horizontal and vertical slide limits allows it to act as a 1D or 2D slider
-   */
-  [RequireComponent(typeof(InteractionBehaviour))]
-  public class InteractionSlider : MonoBehaviour {
-    [Tooltip("The minimum and maximum heights the button can exist at.")]
-    public Vector2 MinMaxHeight = new Vector2(0f, 0.02f);
-    [Tooltip("The height that this button rests at; this value is a lerp in between the min and max height.")]
-    public float RestingHeight = 0.5f;
-    [Tooltip("The minimum and maximum horizontal extents that the slider can slide to.")]
+  
+  /// <summary>
+  /// A physics-enabled slider. Sliding is triggered by physically pushing the slider to its compressed position. 
+  /// Increasing the horizontal and vertical slide limits allows it to act as either a 1D or 2D slider.
+  /// </summary>
+  public class InteractionSlider : InteractionButton {
+
+    [Space, Space]
+    [Tooltip("The minimum and maximum values that the slider reports on the horizontal axis.")]
+    public Vector2 HorizontalValueRange = new Vector2(0f, 1f);
```

Comment:
capitalization (and for rest of file) - https://docs.google.com/document/d/1SsM0Pg469kTzZCkGvPc9MRMId7N447i94BelVp6HAG8/edit